frontend/pages/for-investors.tsx

```diff
@@ -0,0 +1,451 @@
+          onClick={() =>
+            push({
+              pathname: Paths.Discover,
+              // TODO: CHANGE TO FILTER BY PROIORITY LANDSCAPE
+              query: !!categoryColor ? { 'filter[category]': id } : { search: name },
+            })
+          }
+        >
+          <Icon
+            icon={ArrowIcon}
+            className="transition-all duration-500 md:opacity-0 w-15 group-hover:opacity-100 ease"
+          />
+        </Button>
+      </div>
+    </div>
+  );
+
+  return (
+    <>
+      <Head title="About" />
+      <ImpactModal impactModalOpen={impactModalOpen} setImpactModalOpen={setImpactModalOpen} />
+      <LayoutContainer className="bg-background-light">
+        <div className="mb-10 lg:pt-8">
+          <h1 className="font-serif text-3xl font-bold lg:text-6xl text-green-dark">
+            <FormattedMessage defaultMessage="Start having the greatest impact" id="o0+PZ+" />
+          </h1>
+        </div>
+        <div className="mt-8 lg:mt-0 lg:flex lg:flex-col lg:gap-y-10 mb-15 lg:mb-0">
+          <div className="mb-8 lg:flex lg:gap-80 lg:mb-0">
+            <p className="flex-1">
+              <FormattedMessage
+                defaultMessage="HeCo Invest manages a <n>wide range of investment and financing opportunities</n> in various sector categories and priority landscapes for the conservation and development of the <n>Colombian Amazon region</n>."
+                id="NG6Ull"
+                values={{
+                  n: (chunk: string) => <span className="font-semibold">{chunk}</span>,
+                }}
+              />
+            </p>
+            <div className="flex-1"></div>
+          </div>
+          <div className="lg:flex lg:gap-x-80">
+            <div className="flex-1"></div>
+            <p className="flex-1">
+              <FormattedMessage
+                defaultMessage="Thanks to our integration with <n>ARIES</n>, an Artificial Intelligence tool that integrates <n>expert knowledge, Semantics</n> and <n>Machine Reasoning</n> technology, we can <n>estimate a projects’ impact</n> along our four dimensions of interest: <n>Climate, Biodiversity, Water, Community</n> to guide and track your investment decisions."
+                id="LmyoPR"
+                values={{
+                  n: (chunk: string) => <span className="font-semibold">{chunk}</span>,
+                }}
+              />
+            </p>
+          </div>
+        </div>
+      </LayoutContainer>
+
+      <LayoutContainer className="mt-5 lg:mt-32">
+        <div className="flex flex-col lg:py-14 lg:grid lg:grid-cols-2">
+          <div className="flex flex-col justify-center order-2 p-6 pb-10 overflow-hidden text-white lg:py-10 lg:order-1 lg:-mt-28 rounded-b-2xl lg:rounded-3xl lg:rounded-br-none bg-green-dark lg:px-14">
+            <h2 className="font-serif text-3xl font-bold lg:text-4xl xl:text-5xl">
+              <FormattedMessage defaultMessage="30 million" id="m1w8ew" />
+            </h2>
```

Comment:
These 3 figures (“30 million”, “123.000 million” and “10%”) shouldn't be a title. By themselves, they don't mean anything. Use a simple `<div />` instead.